tests/models/registry.py

```diff
@@ -209,8 +214,8 @@ def check_available_online(
                                         trust_remote_code=True),
     "QWenLMHeadModel": _HfExamplesInfo("Qwen/Qwen-7B-Chat",
                                        trust_remote_code=True),
-    "Qwen2ForCausalLM": _HfExamplesInfo("Qwen/Qwen2-7B-Instruct",
-                                        extras={"2.5": "Qwen/Qwen2.5-7B-Instruct"}), # noqa: E501
+    "Qwen2ForCausalLM": _HfExamplesInfo("Qwen/Qwen2.5-0.5B-Instruct",
+                                        extras={"2.5": "Qwen/Qwen2.5-1.5B-Instruct"}), # noqa: E501
```

Comment:
```suggestion
    "Qwen2ForCausalLM": _HfExamplesInfo("Qwen/Qwen2-0.5B-Instruct",
                                        extras={"2.5": "Qwen/Qwen2.5-0.5B-Instruct"}), # noqa: E501
```

I think we should test Qwen2.0 and use 2.5 in extra here.